src/algorithms/telemetry_decoder/gnuradio_blocks/sbas_l1_telemetry_decoder_gs.cc

```diff
@@ -41,20 +42,23 @@
 
 sbas_l1_telemetry_decoder_gs_sptr sbas_l1_make_telemetry_decoder_gs(
     const Gnss_Satellite &satellite,
-    bool dump)
+    const Tlm_Conf &conf)
 {
-    return sbas_l1_telemetry_decoder_gs_sptr(new sbas_l1_telemetry_decoder_gs(satellite, dump));
+    return sbas_l1_telemetry_decoder_gs_sptr(new sbas_l1_telemetry_decoder_gs(satellite, conf));
```

Comment:
The body of legacy examples notwithstanding, in modern C++ explicit calls to `operator new` generally indicate a memory management problem. Modern code should never have `new` or `delete`; use the appropriate type of managing pointer and its associated factory function. Here, consider
```suggestion
    return std::make_shared<sbas_l1_telemetry_decoder_gs>(satellite, conf);
```

Note that @carlesfernandez may have a different opinion, but I don't see a viable path for adding enhancements and supporting archaic environments. This is just my opinion, but I would write all **new** features for current and future platforms/environments, not frozen legacy ones. 

The key is not to break anything in the process, so it may be that new features need to be conditionally enabled so obsolete environments can ignore them. Personally, I'd freeze a release that supports the old environments that we care about and require building from source (HEAD) to be up-to-date for standards and external versions. That's outside the scope of your change, so you don't need to fix it, but you do need to be aware of the tension.